homeassistant/components/emulated_hue/hue_api.py

```diff
@@ -571,7 +571,7 @@ def get_entity_state(config, entity):
 def entity_to_json(config, entity, state):
     """Convert an entity to its Hue bridge JSON representation."""
     entity_features = entity.attributes.get(ATTR_SUPPORTED_FEATURES, 0)
-    if (entity_features & SUPPORT_BRIGHTNESS) or entity.domain != light.DOMAIN:
+    if (entity_features & SUPPORT_BRIGHTNESS) or entity.domain in [climate.DOMAIN, fan.DOMAIN, media_player.DOMAIN, cover.DOMAIN]:
```

Comment:
```suggestion
    if (entity_features & SUPPORT_BRIGHTNESS) or entity.domain in [
        climate.DOMAIN,
        fan.DOMAIN,
        media_player.DOMAIN,
        cover.DOMAIN,
    ]:
```
Your commit cannot be merged because of code style issues; you might want to reformat according to the suggestion to pass the CI check.